packages/debugger/src/handlers/notebook.ts

```diff
@@ -26,10 +32,34 @@ export class NotebookHandler implements IDisposable {
     this._debuggerService = options.debuggerService;
     this._notebookPanel = options.widget;
     this._cellMap = new ObservableMap<EditorHandler>();
+    this.translator = options.translator || nullTranslator;
+    this._trans = this.translator.load('jupyterlab');
 
     const notebook = this._notebookPanel.content;
     notebook.model!.cells.changed.connect(this._onCellsChanged, this);
 
+    this._debuggerService.session?.eventMessage.connect((_, event) => {
+      const session = this._debuggerService.session;
+      const contextSession = this._notebookPanel.sessionContext.session;
+
+      if (!session || !contextSession) {
+        return;
```

Comment:
I think this prevents the overlay from showing it up in the console - is it expected? Again, it is perfectly fine to keep this PR small or to not add this for console - just want to make sure what is intended and what is not.

<img width="989" height="707" alt="Image" src="https://github.com/user-attachments/assets/9df84002-8782-4852-a702-ab053f73275a" />